app/models/appointment_slot_preset.rb

```diff
@@ -1,8 +1,29 @@
 class AppointmentSlotPreset < ActiveRecord::Base
+  include PushUpdatesMinimal
+
+  validate :no_conflict, on: :update
+
   validates :day, numericality: { only_integer: true, greater_than: 0, less_than: 8  }
   validates :hours, numericality: { only_integer: true, greater_than_or_equal_to: 0, less_than: 24 }
   validates :minutes, numericality: { only_integer: true, greater_than_or_equal_to: 0, less_than: 60 }
   validates :quota, numericality: { only_integer: true, greater_than_or_equal_to: 1 }
 
   scope :ascending, -> { order('hours ASC').order('minutes ASC') }
+
+  after_save :push_changes
+  after_destroy :push_changes
+  push_targets [ Channel::STOCK_CHANNEL ]
+
+  private
+
+  def no_conflict
+    conflicts = AppointmentSlotPreset
+      .where({ day: self.day, hours: self.hours, minutes: self.minutes })
+      .where.not({ id: self.id })
+      .count
+
+    if conflicts > 0
```

Comment:
We can use method `.positive?` here instead of `> 0`

Reply:
we can, but I'm curious what's the value 🤔

Reply:
it's just a method of numeric class.

https://ruby-doc.org/core-2.3.0/Numeric.html#method-i-positive-3F

I feel it more readable on why not to use if ruby provides it. 

Again depends on personal choice. :) 

Just thought to mention this method.